src/frontend/screens/MapScreen.js

```diff
@@ -42,6 +43,14 @@ const MapScreen = ({ navigation }: Props) => {
     [navigation, newDraft]
   );
 
+  React.useEffect(() => {
+    deepLinkSetUp(navigation.navigate);
+
+    return () => {
+      deepLinkTakeDown(navigation.navigate);
+    };
+  }, []);
```

Comment:
I'd probably create a hook for this just to isolate the code a little more

Reply:
Good idea!